vita3k/Vita3K.rc

```diff
@@ -1,3 +1,20 @@
+// Vita3K emulator project
+// Copyright (C) 2019 Vita3K team
```

Comment:
```suggestion
// Copyright (C) 2021 Vita3K team
```
all time keep year in current year :)

Reply:
Should I update the copyright year to 2021 only on the source code files I placed a copyright notice on or on every source code file on the repository?

Reply:
hmm i thinks using like some other i have see update copyright on this year for all file project